teku/src/main/java/tech/pegasys/teku/cli/subcommand/ValidatorClientCommand.java

```diff
@@ -0,0 +1,134 @@
+      return 0;
+    } catch (InvalidConfigurationException | DatabaseStorageException ex) {
+      parentCommand.reportUserError(ex);
+    } catch (CompletionException e) {
+      if (Throwables.getRootCause(e) instanceof InvalidConfigurationException) {
+        parentCommand.reportUserError(Throwables.getRootCause(e));
+      } else {
+        parentCommand.reportUnexpectedError(e);
+      }
+    } catch (Throwable t) {
+      parentCommand.reportUnexpectedError(t);
+    }
+    return 1;
+  }
+
+  private TekuConfiguration tekuConfiguration() {
+    return TekuConfiguration.builder()
+        .setNetwork(NetworkDefinition.fromCliArg(networkOptions.getNetwork()))
+        .setInteropGenesisTime(interopOptions.getInteropGenesisTime())
+        .setInteropOwnedValidatorStartIndex(interopOptions.getInteropOwnerValidatorStartIndex())
+        .setInteropOwnedValidatorCount(interopOptions.getInteropOwnerValidatorCount())
+        .setInteropNumberOfValidators(interopOptions.getInteropNumberOfValidators())
+        .setInteropEnabled(interopOptions.isInteropEnabled())
+        .setValidatorKeystoreFiles(validatorOptions.getValidatorKeystoreFiles())
+        .setValidatorKeystorePasswordFiles(validatorOptions.getValidatorKeystorePasswordFiles())
+        .setValidatorKeys(validatorOptions.getValidatorKeys())
+        .setValidatorExternalSignerPublicKeys(
+            validatorOptions.getValidatorExternalSignerPublicKeys())
+        .setValidatorExternalSignerUrl(validatorOptions.getValidatorExternalSignerUrl())
+        .setValidatorExternalSignerTimeout(validatorOptions.getValidatorExternalSignerTimeout())
+        .setGraffiti(validatorOptions.getGraffiti())
+        .setLogColorEnabled(loggingOptions.isLogColorEnabled())
+        .setLogIncludeEventsEnabled(loggingOptions.isLogIncludeEventsEnabled())
+        .setLogIncludeValidatorDutiesEnabled(loggingOptions.isLogIncludeValidatorDutiesEnabled())
+        .setLogDestination(loggingOptions.getLogDestination())
+        .setLogFile(loggingOptions.getLogFile())
+        .setLogFileNamePattern(loggingOptions.getLogFileNamePattern())
+        .setLogWireCipher(loggingOptions.isLogWireCipherEnabled())
+        .setLogWirePlain(loggingOptions.isLogWirePlainEnabled())
+        .setLogWireMuxFrames(loggingOptions.isLogWireMuxEnabled())
+        .setLogWireGossip(loggingOptions.isLogWireGossipEnabled())
+        .setMetricsEnabled(metricsOptions.isMetricsEnabled())
+        .setMetricsPort(metricsOptions.getMetricsPort())
+        .setMetricsInterface(metricsOptions.getMetricsInterface())
+        .setMetricsCategories(metricsOptions.getMetricsCategories())
+        .setMetricsHostAllowlist(metricsOptions.getMetricsHostAllowlist())
+        .setDataPath(dataOptions.getDataPath())
+        .setDataStorageMode(dataOptions.getDataStorageMode())
+        .setDataStorageFrequency(dataOptions.getDataStorageFrequency())
+        .setDataStorageCreateDbVersion(dataOptions.getCreateDbVersion())
```

Comment:
Once #2684 is merged you'll only have data path in `DataOptions` - the others have moved to `DataStorageOptions` but the validator client won't need them anyway.

Reply:
Done :)